app/src/main/java/org/xbmc/kore/ui/widgets/PlaylistsBar.java

```diff
@@ -0,0 +1,175 @@
+        LayoutInflater inflater = (LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+        inflater.inflate(R.layout.playlist_bar, this);
+
+        setStyle(context);
+
+        for(int i = 0; i < getTabCount(); i++) {
+            disableTab(i);
+        }
+    }
+
+    public TabLayout.Tab getTab(String playlistType) {
+        return getTabAt(getTabPositionForType(playlistType));
+    }
+
+    public void selectTab(String playlistType) {
+        getTabAt(getTabPositionForType(playlistType)).select();
+    }
+
+    public void setHasPlaylistAvailable(String playlistType, boolean playlistAvailable) {
+        if (playlistAvailable) {
+            enableTab(getTabPositionForType(playlistType));
+        } else {
+            disableTab(getTabPositionForType(playlistType));
+        }
+    }
+
+    private Runnable runnable;
+
+    public void setIsPlaying(final String playlistType, final boolean isPlaying) {
+        handler.removeCallbacks(runnable);
+
+        runnable = new Runnable() {
+            @Override
+            public void run() {
+                if (isPlaying)
+                    getTabAt(getTabPositionForType(playlistType)).getIcon().setColorFilter(highlightColor, PorterDuff.Mode.SRC_ATOP);
+                else
+                    getTabAt(getTabPositionForType(playlistType)).getIcon().setColorFilter(defaultColor, PorterDuff.Mode.SRC_ATOP);
+            }
+        };
+
+        handler.postDelayed(runnable, 1000);
+    }
+
+    public String getTypeForTabPosition(int tabPosition) {
+        switch (tabPosition) {
+            case 0:
+                return PlaylistType.GetPlaylistsReturnType.VIDEO;
+            case 1:
+                return PlaylistType.GetPlaylistsReturnType.AUDIO;
+            case 2:
+                return PlaylistType.GetPlaylistsReturnType.PICTURE;
+            default:
+                return PlaylistType.GetPlaylistsReturnType.VIDEO;
+        }
+    }
+
+    public String getSelectedPlaylistType() {
+        return getTypeForTabPosition(getSelectedTabPosition());
+    }
+
+    public void setOnPlaylistSelectedListener(final OnPlaylistSelectedListener onPlaylistSelectedListener) {
+        addOnTabSelectedListener(new OnTabSelectedListener() {
+            @Override
+            public void onTabSelected(Tab tab) {
+                onPlaylistSelectedListener.onPlaylistSelected(getTypeForTabPosition(tab.getPosition()));
+            }
+
+            @Override
+            public void onTabUnselected(Tab tab) {
+                onPlaylistSelectedListener.onPlaylistDeselected(getTypeForTabPosition(tab.getPosition()));
+            }
+
+            @Override
+            public void onTabReselected(Tab tab) {
+
+            }
+        });
+    }
+
+    private void enableTab(int tabPosition) {
+        getTabAt(tabPosition).getIcon().setAlpha(255);
+    }
+
+    private void disableTab(int tabPosition) {
+        getTabAt(tabPosition).getIcon().setAlpha(127);
```

Comment:
Somehow this is interfering with the same icon on other places. The movies/music/pictures icons on the main remote shortcut bar and on the navigation drawer also become dimmed. Android seems to be caching them and using the same copy on all places.

Reply:
I am unable to reproduce this on an emulator running API 15 and also tested on API 28. On what device and Android version does this occur?

Reply:
Xperia XZ1, with Android Pie. I'll test on Oreo later, meanwhile, here are some screenshots showing the issue
![screenshot_20190223-192708](https://user-images.githubusercontent.com/6040384/53290827-81e45d00-37a1-11e9-8392-d1b5114e8aa8.png)
![screenshot_20190223-192716](https://user-images.githubusercontent.com/6040384/53290828-8446b700-37a1-11e9-913a-312ff3b2cef8.png)

Reply:
Found it. By default drawable resources share their state. To fix this I made the icons for the playlistbar mutable.